docs/build.gradle

```diff
@@ -52,6 +52,9 @@ testClusters.integTest {
   if (singleNode().testDistribution == DEFAULT) {
     setting 'xpack.license.self_generated.type', 'trial'
     setting 'indices.lifecycle.history_index_enabled', 'false'
+    if (BuildParams.isSnapshotBuild() == true) {
```

Comment:
now that I am seeing this, I may have to change how I've done this in my PRs.

I feel it is safe to always set the system property in test clusters. I'll change all of them at a later point